src/main/java/com/zendesk/maxwell/schema/SchemaStoreSchema.java

```diff
@@ -99,6 +99,10 @@ public static void upgradeSchemaStoreSchema(Connection c) throws SQLException, I
 			performAlter(c, "alter table `schemas` add column deleted tinyint(1) not null default 0");
 		}
 
+		if ( !getTableColumns("schemas", c).containsKey("gtid_set") ) {
+			performAlter(c, "alter table `schemas` add column gtid_set varchar(255)");
```

Comment:
Use 'Text' seems a big jump since it's not that big and TEXT is stored out of line of the row.

MySQL can allow big varchar (up to 64K bytes for the whole row size), do you want to use 4K for now?

In the normal situation, for one server, gtid_set probably takes 30-40 chars.